### PR TITLE
ddev-cypress: Cypress configured with ddev

### PR DESCRIPTION
**Pull request by gauravaxl**

### CODE REVIEW

File: .ddev/docker-compose.cypress.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+services:
+  cypress:
+    image: cypress/included:12.6.0
+    container_name: ddev-${DDEV_SITENAME}-cypress
+    labels:
+      com.ddev.site-name: ${DDEV_SITENAME}
+      com.ddev.approot: ${DDEV_APPROOT}
+    user: "1000:1000"
```
Comment (hussainweb) [inline .ddev/docker-compose.cypress.yaml:8]: Does it break without this? As per [this](https://github.com/tyler36/ddev-cypress/issues/26#issuecomment-1588302459), it is already taken care of.

**Comment by hussainweb** [comment]
Another thing @gauravaxl, can you write instructions to run this in the README.md file?

File: .ddev/docker-compose.cypress.yaml (hunk 1 above)
Comment (gauravaxl) [reply .ddev/docker-compose.cypress.yaml]: No, It doesn't break. I have removed the code.

File: .ddev/docker-compose.cypress.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    tty: true
+
+    environment:
+      - CYPRESS_baseUrl=${DDEV_PRIMARY_URL}
+      - DISPLAY=host.docker.internal:0
+
+    volumes:
+      # Mount the project root to Cypress's project point
+      - ${DDEV_APPROOT}/web/themes/custom/contribtracker:${DDEV_APPROOT}/web/themes/custom/contribtracker
```
[the remark this answers is not in the document]
Comment (hussainweb) [reply .ddev/docker-compose.cypress.yaml:15]: Yes, I understand and you have changed the working directory for that. I am talking about the line that I highlighted:

```
      # Mount the project root to Cypress's project point
      - ${DDEV_APPROOT}/web/themes/custom/contribtracker:${DDEV_APPROOT}/web/themes/custom/contribtracker
```

That can be the default, right? If you mount the root directory, Cypress files will be available in this location anyway.

[the remark this answers is not in the document]
Comment (hussainweb) [reply .ddev/docker-compose.cypress.yaml:15]: This change goes hand in hand with the change in line 26 to `working_dir`. In this case, `working_dir` should be `/e2e/web/themes/custom/contribtracker`.

[the remark this answers is not in the document]
Comment (gauravaxl) [reply .ddev/docker-compose.cypress.yaml]: Thanks, It worked. I have updated the path

**Comment by zeshanziya** [comment]
@hussainweb Please review the changes. 

I tried updating the cypress image to version 13. but it seems video recording not working with that version. I will check whats the issue and update the cypress image to latest version.

**Review by hussainweb — APPROVED** [review]

**Pull request merged**